internal/exec/yaml_func_terraform_output.go

```
@@ -15,26 +15,42 @@
+	if partsLen == 3 {
+		component = strings.TrimSpace(parts[0])
+		stack = strings.TrimSpace(parts[1])
+		output = strings.TrimSpace(parts[2])
+	} else if partsLen == 2 {
+		component = strings.TrimSpace(parts[0])
+		stack = currentStack
+		output = strings.TrimSpace(parts[1])
+		u.LogTrace(cliConfig, fmt.Sprintf("Atmos YAML function `%s` is called with two parameters 'component' and 'output'. "+
+			"Using the current stack '%s' as the 'stack' parameter", input, currentStack))
+	} else {
+		err := errors.New(fmt.Sprintf("invalid number of arguments in the Atmos YAML function: %s", input))
```

golangci-lint (1.62.2)

S1028: should use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...))

(gosimple)